engine/source/materials/mat/mat004/m4law.F

```diff
@@ -50,34 +50,35 @@ SUBROUTINE M4LAW(
 C   D u m m y   A r g u m e n t s
 C-----------------------------------------------
       INTEGER, INTENT(IN) :: JTHE
+      INTEGER, INTENT(IN) :: IEOS
       INTEGER MAT(MVSIZ),NEL
 C     REAL
       my_real
      .   PM(NPROPM,*), OFF(MVSIZ), SIG(NEL,6), EPXE(MVSIZ),
-     .   TEMPER(MVSIZ),SSP(MVSIZ), D1(MVSIZ),    D2(MVSIZ), 
+     .   VOL(NEL),SSP(MVSIZ), D1(MVSIZ),    D2(MVSIZ), 
```

Comment:
@MaciekWronski 

INTENT statement is missing for this 'new' argument

Reply:
the INTENT statements are already added, except if I missed something